### DISCUSSION TITLE
Labels for notes

### DISCUSSION THREAD

**Discussion by AGNMM18** [CareTogether/CareTogetherCMS#407] [Ideas]
Enhancement request: Can we have a list of labels for notes, so we can easily tell what type of note we are reading?

**Comment by LarsKemmann** [comment]
Intake
Meeting w/ parent
Contact w/ host family
Goal planning session
Contact w/ service provider
Text exchanged w/ parent
... (managed via organization configuration)

Select labels via multi-option tag controls (https://mui.com/material-ui/react-chip/#clickable-and-deletable).
